source/componentsDowngrade.ts

```diff
@@ -67,6 +69,7 @@ export function downgradeComponentsToAngular1(upgradeAdapter: UpgradeAdapter) {
 
 	upgradeAdapter.addProvider(DEFAULT_THEME_PROVIDER);
 	upgradeAdapter.addProvider(FormService);
+	upgradeAdapter.addProvider(WindowService);
```

Comment:
There are actually two steps to downgrading an angular 2 provider:
1: Add the provider to the upgrade adapter. This makes it available for anyone to inject and essentially takes the place of providing it on the ng2 app (which we don't have)
2: Register the downgraded provider to angular 1, via:
`componentsDowngradeModule.factory(windowServiceName, upgradeAdapter.downgradeNg2Provider(WindowService));`
The first one is something we will be doing for all of our singleton services until we get rid of Angular 2, which is why the form service is adding itself as a provider here. (I was confused about what you were asking on Slack)
You'll have to complete step 2 for the window and document services